landing-pages/site/content/en/ecosystem/_index.md

```diff
@@ -53,7 +53,7 @@ If you would you like to be included on this page, please reach out to the [Apac
 
 [Self-Managed Airflow via CNDI](https://github.com/polyseam/cndi) - Toolkit for deploying Airflow Kubernetes clusters, with support for AWS, GCP, Azure, VMWare, Bare-Metal, and even multi/hybrid cloud support. See [docs](https://docs.cndi.run) for more details.
 
-[Self-managed Airflow on Amazon EKS](https://github.com/awslabs/data-on-eks/tree/main/schedulers/terraform/managed-airflow-mwaa) - Self-managed Airflow on Amazon EKS provides a guide for deploying self-managed Apache Airflow on [Amazon EKS](https://aws.amazon.com/eks/) with [Terraform](https://www.terraform.io/) using Data on EKS Blueprints with the [Terraform Data add-ons](https://registry.terraform.io/modules/aws-ia/eks-data-addons/aws/latest) module.
+[Self-managed Airflow on Amazon EKS](https://github.com/awslabs/data-on-eks/tree/main/schedulers/terraform/self-managed-airflow) - Self-managed Airflow on Amazon EKS provides a guide for deploying self-managed Apache Airflow on [Amazon EKS](https://aws.amazon.com/eks/) with [Terraform](https://www.terraform.io/) using Data on EKS Blueprints with the [Terraform Data add-ons](https://registry.terraform.io/modules/aws-ia/eks-data-addons/aws/latest) module.
```

Comment:
Replace the first link in this line with website doc please https://awslabs.github.io/data-on-eks/docs/blueprints/job-schedulers/self-managed-airflow

Reply:
For MWAA below we show 2 links.. one for github and one for the doc site. Do you want the same or just have the doc site?